ui/app/adt/views/wardList.html

```diff
@@ -1,15 +1,16 @@
 <div class="ward-details-container">
+    <input type="text" ng-model="searchText" placeholder="Search..."/>
```

Comment:
if possible, can we have a translation key for placeholder as well ?

Reply:
Done @kavitha-sundararajan